lib/avro_turf/schema_registry.rb

```diff
@@ -18,7 +18,7 @@ def fetch(id)
 
   def register(subject, schema)
     data = post("/subjects/#{subject}/versions", body: {
-      schema: schema
+      schema: schema.to_s
```

Comment:
For reference, see the definition of [`Avro::Schema#to_s`](https://github.com/apache/avro/blob/315d842148d57590a58fafecf6e5ea378e9e0d74/lang/ruby/lib/avro/schema.rb#L178):

```ruby
    def to_s
      MultiJson.dump to_avro
    end
```